Expensify/App
# [User Settings] Implement new Profile view

<If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

### Details
- Set up new Profile route, uses the personalDetails tied to the login or the report (as a fallback)
- Redo MultipleAvatars to be able to be wrapped without getting messed up

### Fixed Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is fixing>
Fixes https://github.com/Expensify/Expensify.cash/issues/1492

### Tests
1. start a chat with a verified user and click their avatar in the header to get routed to see their profile info
2. start a chat with an unverified user and click their avatar in the header to get routed to see their profile info and minimally see their avatar & email/login
3. start a chat with multiple users and confirm that clicking their avatar does nothing

### Tested On


### Video Walkthrough


https://user-images.githubusercontent.com/12075600/109588212-3dd97080-7abd-11eb-8f6b-bff8ce58152f.mov



#### Web

![Screen Shot 2021-03-01 at 6 29 29 PM](https://user-images.githubusercontent.com/12075600/109588301-5c3f6c00-7abd-11eb-91ac-dea68f872b0e.png)

#### Mobile Web

<img width="494" alt="Screen Shot 2021-03-01 at 6 41 46 PM" src="https://user-images.githubusercontent.com/12075600/109588582-d66ff080-7abd-11eb-8034-0841709de181.png">

#### Desktop

<img width="1312" alt="Screen Shot 2021-03-01 at 6 27 44 PM" src="https://user-images.githubusercontent.com/12075600/109588316-63667a00-7abd-11eb-8643-40a772fc1ac3.png">

#### iOS

<img width="494" alt="Screen Shot 2021-03-01 at 6 24 07 PM" src="https://user-images.githubusercontent.com/12075600/109588363-7711e080-7abd-11eb-9eaa-646cfeef1b56.png">

#### Android

<img width="418" alt="Screen Shot 2021-03-01 at 6 26 40 PM" src="https://user-images.githubusercontent.com/12075600/109588416-8a24b080-7abd-11eb-816a-ad16785117f5.png">
